src/DotNetWorker.Core/FunctionMetadata/IRetryOptions.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Core.FunctionMetadata
+{
+    /// <summary>
+    /// Function execution retry policy to use on invocation failures.
+    /// </summary>
+    public interface IRetryOptions
```

Comment:
For the exponential backoff, do we need to also include a backoff coefficient? Or is that something we have historically not included?

Reply:
Historically has not been included - here's the related webjobs type [here](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.Abstractions/Description/RetryOptions.cs).